wiki/en/en-Software-Manual.md

```diff
@@ -57,13 +53,13 @@ Make sure not to clip the input signal to avoid distortions of the audio signal
 
 ![Chat dialog](https://user-images.githubusercontent.com/20726856/97360717-d7d6a580-189e-11eb-84db-80536e17dc30.png)
 
-The chat text entered in that dialog is sent to
-all connected clients. If a new chat message arrives and the Chat dialog is not already open, it will
+The chat text entered in that dialogue is sent to
```

Comment:
'Dialog' seems to be the common spelling when referring to the GUI item, though strictly speaking 'dialogue' would be more formally correct. Here we're talking about the chat box so 'dialogue' might even be confusing.